regression/goto-harness/pointer-to-array-function-parameters-multi-arg-wrong/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+test.c
+--harness-type call-function --function is_prefix_of --treat-pointer-as-array string --treat-pointer-as-array prefix --associated-array-size string:string_size --associated-array-size prefix:prefix_size --max-array-size 5
+EXIT=10
+SIGNAL=0
```

Comment:
anchoring

Reply:
All of these got handled at the latest push :)